components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -2158,11 +2217,14 @@ private int handleStartIndexEqualsNULL(Integer startIndex) {
         // EnableFilteringEnhancements' properties in identity.xml or domain name embedded in the filter attribute
```

Comment:
We should use multiline comments if the comments spans over more than a single line.

Reply:
This is not done by this PR.